controllers/postController.js

```diff
@@ -335,105 +372,109 @@ const postLike = async (req, res) => {
       },
       {
         $group: {
-          _id: { $dateToString: { date: "$createdAt", format: "%Y.%m.%d", timezone: "+09:00" } },
+          _id: {
+            $dateToString: {
+              date: dateFilter,
+              format: "%Y.%m.%d",
+              timezone: "+09:00",
+            },
+          },
           likeCount: { $sum: "$likeCount" },
+          commentCount: { $sum: "$commentCount" },
         },
       },
       { $addFields: { date: "$_id" } },
       { $project: { _id: 0 } },
     ]);
 
-    if (postLikeResultList.length < DAY_OF_WEEK) {
-      let index = 0;
-
-      while (index < DAY_OF_WEEK) {
-        const targetDate = new Date(cursorStartDate);
-        targetDate.setDate(targetDate.getDate() + index);
-        const transformedTargetMonth =
-          (targetDate.getMonth() + 1).toString().length === 1
-            ? (targetDate.getMonth() + 1).toString().padStart(2, "0")
-            : (targetDate.getMonth() + 1).toString();
-        const transformedTargetDate =
-          targetDate.getDate().toString().length === 1
-            ? targetDate.getDate().toString().padStart(2, "0")
-            : targetDate.getDate().toString();
-        const targetDateString = `${targetDate.getFullYear()}.${transformedTargetMonth}.${transformedTargetDate}`;
-
-        const hasTargetDate = postLikeResultList
+    if (reactionCountListByPeriod.length < periodLength) {
+      let addedDate = 0;
+      let periodUnit = period === PERIOD.MONTHLY_WEEKLY ? 7 : 1;
+
+      while (addedDate < DAY_OF_WEEK) {
```

Comment:
```suggestion
      while (addedDate < periodLength) {
```
위 `postCount` 함수 처럼 `periodLength`와 비교해야 조회할 기간의 모든 데이터가 잘 들어갈 수 있어보입니다!

Reply:
해당 부분 수정하는 거 놓쳤네요..! 매의 눈으로 리뷰해주셔서 감사합니다! ☺️